Content.Server/Access/Systems/IdCardSystem.cs

```diff
@@ -26,6 +37,29 @@ private void OnInit(EntityUid uid, IdCardComponent id, ComponentInit args)
             UpdateEntityName(uid, id);
         }
 
+        private void OnMicrowaved(EntityUid uid, IdCardComponent component, BeingMicrowavedEvent args)
+        {
+            if (TryComp<AccessComponent>(uid, out var access))
+            {
+                // If they're unlucky, brick their ID
+                if (_random.Prob(0.25f))
+                {
+                    _popupSystem.PopupEntity(Loc.GetString("id-card-component-microwave-bricked", ("id", uid)),
+                        uid, Filter.Pvs(uid));
+                    access.Tags.Clear();
+                }
+                else
+                {
+                    _popupSystem.PopupEntity(Loc.GetString("id-card-component-microwave-safe", ("id", uid)),
+                        uid, Filter.Pvs(uid));
+                }
+
+                // Give them a wonderful new access to compensate for everything
+                var random = _random.Pick(_prototypeManager.EnumeratePrototypes<AccessLevelPrototype>().ToArray());
+                access.Tags.Add(random.ID);
```

Comment:
Currently (on saltern, probably others), all it takes is HOP access to get into the office and gain any access you want.

I guess there's a lot of RNG involved and you still need to get in without being seen, so maybe its fine? But then again, maybe consider restricting HOP somehow?

Reply:
well, it's basically impossible to figure out which access you got without testing it

i can just give it a pretty low chance to completely brick it or something

Reply:
Ah yeah I forgot about the fact that there's no way to see access, and that someone would have to very conspicuously keep walking from the kitchen to the hops office and trying to get in. Probably fine as it is.